ironfish/src/rpc/routes/wallet/rescanAccount.ts

```diff
@@ -46,20 +46,20 @@ routes.register<typeof RescanAccountRequestSchema, RescanAccountResponse>(
 
       let fromHash = undefined
       if (request.data.from && request.data.from > GENESIS_BLOCK_SEQUENCE) {
-        const header = await node.chain.getHeaderAtSequence(request.data.from)
+        const response = await node.wallet.chainGetBlock({ sequence: request.data.from })
```

Comment:
Originally we were fetching the header, but now this is going to return the full block with transactions. Should we only use the header here?

Reply:
Do we have an RPC call for that? I just used this because I thought it was the only call we had and didn't want to make add another endpoint for this specific use case.

Reply:
Discussed. This RPC call specifically is a one-off call so bandwidth is not as big of an issue so we'll use the getBlock call for now.